ddtrace/internal/ci_visibility/recorder.py

```diff
@@ -353,7 +375,7 @@ def _start_service(self):
             self.tracer.configure(settings={"FILTERS": tracer_filters})
         if self._git_client is not None:
             self._git_client.start(cwd=_get_git_repo())
-        if self.test_skipping_enabled() and self._test_suites_to_skip is None:
+        if self.test_skipping_enabled() and (not self._tests_to_skip and self._test_suites_to_skip is None):
```

Comment:
maybe
```
if self.test_skipping_enabled() and not (self._tests_to_skip or self._test_suites_to_skip) ?
```

Reply:
Yes, this is more accurate, but since we change `_test_suites_to_skip` in `_fetch_tests_to_skip()`  from `None` to `[]` it is enough to know wether this initialization function was called